roles/overpass-api/tasks/set_up_database.yml

```diff
@@ -30,3 +31,4 @@
     enabled: yes
     daemon_reload: yes
     no_block: yes
+  when: service.changed
```

Comment:
Isn't this automatically handled by ansible ?

Tasks are supposed to not do anything if not necessary.

Reply:
It is, but this service is stop and inhibed after its first run. Ansible see it stopped and then try to restart it.

Reply:
Ok. It is a strange behaviour, but that's fine.